test/config.test.json

```diff
@@ -32,6 +32,9 @@
         "PASSWORD": "password",
         "USE_TLS": false
     },
+    "DATA": {
+        "ROOT_ORGANISATION_ID": "id-of-the-root-organisation"
```

Comment:
This will break as soon as we introduce a check for UUID. Please generate a random one (websites for that) and use it